spec/models/skill_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Skill, type: :model do
+  let(:role) { FactoryGirl.create(:role) }
+  let(:user) { FactoryGirl.create(:user, role_id: role.id) }
+  let(:skill) { FactoryGirl.create(:skill, user_id: user.id) }
+
+  it 'invalid if user.id value is not equal to skill.id' do
+    is_expected.not_to be_valid if skill.id != user.id
+  end
+
+  it 'invalid if experience value is not an integer in range' do
+    (50..10_000).each do |val|
```

Comment:
Realy??? Run the test 10 000 times?

Reply:
Why?? You can just take one value from the range and one out of the range (example `10 0001`)